README.md

```diff
@@ -1,20 +1,50 @@
-# UGM2023
-Materials for the [MDAnalysis User Group Meeting 2023](https://www.mdanalysis.org/pages/ugm2023/).
+# MDAnalysis UGM 2023
+The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
 
-## Code of conduct
+## When and Where
+### **Main Meeting (27th & 28th September)**
 
-All members of the MDAnalysis community and participants in this UGM are expected to abide by the MDAnalysis [Code of Conduct](https://www.mdanalysis.org/pages/conduct/).
+[Faculty of Sciences of the University of Lisbon](https://www.ulisboa.pt/en/unidade-organica/faculty-sciences) (ULisboa) Auditorium, Building C1
+Campo Grande, 1749-016 Lisbon, Portugal
+
+*A campus map (ULisboa Map.pdf) is included in this repo.* 
```

Comment:
can we make this a link to the pdf?

Reply:
I think I've added the links - hopefully they will work okay

Reply:
Both work great on my end thanks!